### ISSUE TITLE
Content missing after heading - SiteImprove - Banner Page Title Replacement

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa801#8504]
# Problem 

Siteimprove flags banner blocks being used as page title replacement as "Content missing after heading" issue.  

> Headings are used to structure content in a hierarchy. This helps users to know where to start and how content sections relate to one another.
> 
> Content should appear as expected including between headings of the same level and after the last section or page heading.
> 
> Headings structure content in an accessible way. They should be organized in a logical hierarchy so that screen reader and keyboard-only users can easily understand and navigate content.
> 
> Content should appear as expected including between headings of the same level and after the last section or page heading. If content is missing then the user does not know if the information is hidden, or if a heading level was applied incorrectly.

<img width="629" alt="Image" src="https://github.com/user-attachments/assets/04118b2b-b9b4-4b95-91f7-8c6c40aa72ef" />

![Image](https://github.com/user-attachments/assets/21656a97-314d-4989-ac8f-b60e8b21e48d)

# Proposed Solution

- Add a variation of https://github.com/uiowa/uiowa/issues/4543, where we allow:
  -  Headline style adjustment
  -  Banner alignment of the banner title `<h1>`
  -  Visually hide breadcrumbs

**Comment by bspeare** [comment]
Created a new issue that would solve for this https://github.com/uiowa/uiowa/issues/8697.

**Comment by bspeare** [comment]
CLAS has reported this issue and they would benefit from https://github.com/uiowa/uiowa/issues/8697 as well.

**Comment by bspeare** [comment]
Melanie brought this issue up in Office Hours regarding https://engineering.uiowa.edu/ece.  They are trying to move from the default page design of https://engineering.uiowa.edu/bme to a page design using the banner block to give the feel that the Department site is a mini-site within the College site.  For them, using the banner block and menu block with a black background creates this distinction but it also creates this Siteimprove warning.  

**Before:**

<img width="1690" height="1257" alt="Image" src="https://github.com/user-attachments/assets/461cc35b-dc2d-4b33-9e72-71c11a3c2ff1" />

**After:**

<img width="1654" height="1275" alt="Image" src="https://github.com/user-attachments/assets/92d0ae47-8114-4c23-97ce-a636cd48aebe" />

**Comment by m-blaisdell** [comment]
@bspeare I see that the SI issue is tagged "S" which is a best practice. Of course, we want to follow best practices, but I don't see that the issue is showing up in the compliance > Title II tool, so technically it's not a legal compliance issue.

At the very least, I recommend their navigation bar color be offset from their banner for UX, but that's not going to fix the no content after header issue. 

I think if Issue #8697 is prioritized, it would allow for them to have the same effect and be within best practices. Either way, they're going to have to go back to these pages and update once/if that capability is offered. 

I'm not entirely sure I understand the question, though -- are they asking if they should revert or stop updating more pages to this style?

**Comment by bspeare** [comment]
Thanks @m-blaisdell, I appreciate the writeup!  I think the main issue that I missed the first time looking at this is that visually, the page title is being replaced by the department name in the banner for every page.  The visual hierarchy makes the department name most prominent and "Undergraduate program" becomes a content subheading afterthought.  This creates confusion over what the page title is when you land on this page and sibling pages and disrupts the accessibility/readability and of the content outline which is not caught in the SiteImprove scan unfortunately.   I agree that they should follow the model of https://tippie.uiowa.edu/iowa-mba/admissions where the banner "pre-title" would say "Electrical and Computer Engineering" and the h2 would display the actual page title to give the page a visual hierarchy.  

- https://engineering.uiowa.edu/ece/ece-undergraduate-program

The content outline becomes this:

<img width="497" height="368" alt="Image" src="https://github.com/user-attachments/assets/f2bcd01b-9909-445a-a271-54e3c3023d9a" />

When it should be more like:

<img width="475" height="271" alt="Image" src="https://github.com/user-attachments/assets/831389ec-0852-46c4-90e5-8c21b8ad3adb" />

If you see the "Electrical and Computer Engineering" appearing twice in the content outline it is coming from the h2 from the banner and the menu block creating more accessibility issues. 

<img width="1843" height="1164" alt="Image" src="https://github.com/user-attachments/assets/b783b957-65b5-4062-a7e8-3f96b026dacd" />
